Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.cs

```diff
@@ -13,6 +13,31 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// </summary>
     public sealed partial class MarkdownTextBlock : Control, ILinkRegister, IImageResolver, ICodeBlockResolver
     {
+
+        private long FontSizePropertyToken { get; set; }
```

Comment:
Very last one - (everything else is good!) - the toolkit follows the code guidelines of [.NET Core Foundational libraries](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), so ideally all of these private properties should be _camelCase like `private long _fontSizePropertyToken { get; set; }`